components/template/Templates.vue

```diff
@@ -79,15 +82,33 @@ export default {
     return {
       searchTerm: '',
       showFullDescription: false,
+      showHandleButton: false,
     }
   },
+  watch: {
+    activeCategory: {
+      immediate: true,
+      handler() {
+        this.$nextTick(() => {
+          this.checkDescriptionOverflow()
+        })
+      },
+    },
+  },
   methods: {
     handleSearch(searchTerm) {
       this.searchTerm = searchTerm
     },
     toggleDescription() {
       this.showFullDescription = !this.showFullDescription
     },
+    checkDescriptionOverflow() {
+      const description = this.$refs.description
+      if (description) {
+        this.showHandleButton =
+          description.scrollHeight > description.clientHeight
+      }
+    },
```

Comment:
What are we trying to do here?

Reply:
If the description is one line only we don't need to show the Show more btn

Reply:
Is there any other way to achieve this. Instead of `scrollHeight` and `clientHeight`?
There could be a way to achieve this using html and css.

Reply:
I think we still need JS to conditionally check if the text is more than one line or not.
Again, if more than one line, show the button else not